extensions/db/src/main/java/org/mule/extension/db/internal/domain/metadata/DbInputMetadataResolver.java

```diff
@@ -59,7 +46,7 @@ public MetadataType getInputMetadata(MetadataContext context, String query)
     List<InputQueryParam> inputParams = queryTemplate.getInputParams();
     // No metadata when no input parameters
     if (inputParams.size() == 0) {
-      return typeBuilder.nullType().build();
+      return typeBuilder.voidType().build();
```

Comment:
for me this is NullType, because we should communicate that they can ignore the parameter (ie, leave it null). For me a parameter with "no type" (void) is not a valid metadata description.